src/api/certificateAuthority.js

```diff
@@ -1,7 +1,7 @@
 import Q from "q";
 import logger from "winston";
 import pem from "pem";
-import { Keystore, Certificate } from "../model/keystore";
+import { KeystoreModelAPI, CertificateAPI } from "../model/keystore";
```

Comment:
So this one can stay as is, but the other ones will need to be updated to resemble the same format.